modules/qna/src/views/full/index.tsx

```diff
@@ -176,56 +188,94 @@ const QnAList: FC<Props> = ({
     }
   ]
 
-  if (!isLite) {
-    buttons.push(
-      {
-        icon: 'export',
-        disabled: !items.length,
-        onClick: startDownload,
-        tooltip: noItemsTooltip || lang.tr('exportToJson')
-      },
-      {
-        icon: 'import',
-        onClick: () => setShowImportModal(true),
-        tooltip: lang.tr('importJson')
-      }
-    )
-  }
+  buttons.push(
+    {
+      icon: 'export',
+      disabled: !items.length,
+      onClick: startDownload,
+      tooltip: noItemsTooltip || lang.tr('exportToJson')
+    },
+    {
+      content: (
+        <span className={style.customBtn}>
+          <Icon icon={'import' as IconName} />
+          <input
+            type="file"
+            onChange={async e => {
+              if ((e.target as HTMLInputElement).files) {
+                await askUploadOptions((e.target as HTMLInputElement).files[0])
+              }
+            }}
+          />
+        </span>
+      ),
```

Comment:
You don't need to put the input field next to the button. Instead, move the field somewhere in the page, then use a reference to it to trigger the click event

```js
const uploadRef = useRef()

{
  onClick: () => uploadRef?.current?.click()
}

// Put this somewhere in the file
<input
  type="file"
  ref={uploadRef}
            onChange={async e => {
              if ((e.target as HTMLInputElement).files) {
                await askUploadOptions((e.target as HTMLInputElement).files[0])
              }
            }}
          />
```